### DISCUSSION TITLE
RadioLib with SX127X, RTL_433 and OpenMQTTGateway

### DISCUSSION THREAD

**Discussion by 1technophile** [jgromes/RadioLib#672] [Show and tell]
@NorthernMan54 and I would like to thank the [contributors ](https://github.com/jgromes/RadioLib/graphs/contributors) of RadioLib.

**Thanks a lot for sharing this library.**

It is integrated into [rtl_433_ESP](https://github.com/NorthernMan54/rtl_433_ESP) and enables to propose a 433Mhz decoding solution for OOK 433Mhz devices. It can be installed on [Heltec SX127X](https://heltec.org/project/wifi-lora-32/) and [LILYGO® LoRa32 V2.1_1.6.1 433](https://www.lilygo.cc/products/lora3?variant=42476923879605)

![image](https://user-images.githubusercontent.com/12672732/214589384-69699051-65fe-4147-aff4-97afce2955ed.png)

Available with [OpenMQTTGateway](https://docs.openmqttgateway.com/)

Featured below:
https://www.youtube.com/watch?v=H-JXWbWjJYE
https://hackaday.com/2023/01/13/arduino-library-brings-rtl_433-to-the-esp32/
https://www.cnx-software.com/2023/01/14/esp32-board-with-lora-433-mhz-sensors/
https://www.rtl-sdr.com/rtl_433-ported-to-esp32-microcontrollers-with-cc1101-or-sx127x-transceiver-chips/

**Comment by jgromes** [comment]
And thank you for choosing RadioLib in your projects!

For me at least, it's amazing to see other projects finding RadioLib useful, and it's great to see how far it evolved over time.
